pages/deployment/[uuid].tsx

```diff
@@ -181,11 +187,11 @@ export default function Deployment() {
           </Box>
           <Box sx={{ marginBottom: 3 }} />
 
-          {tab === 'overview' && <DeploymentOverview version={deployment} use={'DEPLOYMENT'} />}
+          {tab === 'overview' && <DeploymentOverview version={deployment} use='DEPLOYMENT' />}
 
           {tab === 'compliance' && <ComplianceFlow initialElements={complianceFlow} />}
 
-          {tab === 'build' && <TerminalLog logs={deployment!.logs} title='Deployment Build Logs' />}
+          {tab === 'build' && <TerminalLog logs={deployment?.logs} title='Deployment Build Logs' />}
```

Comment:
This isn't a valid transformation

Reply:
I think again this only renders if deployment exists, so a Type Guard can be used to ignore the `!` or `?` entirely.